src/main/java/de/zalando/aruha/nakadi/exceptions/DuplicatedSubscriptionException.java

```diff
@@ -0,0 +1,15 @@
+package de.zalando.aruha.nakadi.exceptions;
+
+import javax.ws.rs.core.Response;
+
+public class DuplicatedSubscriptionException extends NakadiException {
```

Comment:
I don't think that we will have too many entities in Nakadi to have a generic Exception. For now I'm fine with separate exceptions.